project/AppDependencies.scala

```diff
@@ -7,13 +7,10 @@ object AppDependencies {
   val compile: Seq[ModuleID] = Seq(
     filters,
     ws,
-    "uk.gov.hmrc"       %% "microservice-bootstrap" % "10.4.0",
+    "uk.gov.hmrc"       %% "bootstrap-play-25"      % "5.3.0",
     "uk.gov.hmrc"       %% "domain"                 % "5.6.0-play-25",
-    "org.json4s"        %% "json4s-jackson"         % "3.6.7",
     "uk.gov.hmrc"       %% "time"                   % "3.6.0",
-    "uk.gov.hmrc"       %% "json-encryption"        % "4.4.0-play-25",
     "com.github.fge"     % "json-schema-validator"  % "2.2.6",
-    "org.json4s"        %% "json4s-native"          % "3.5.5",
     "uk.gov.hmrc"       %% "auth-client"            % "2.31.0-play-25",
```

Comment:
Should we upgrade the version of auth-client to 3.0.0-play-25 and domain to 5.9.0-play-25 as well?
Or any other library  to the latest version?

Reply:
We'll upgrade them as part of play26, we have to upgrade all the libraries then anyway